code/mod_table/mod_table.cpp

```diff
@@ -48,6 +48,24 @@ SCP_string Window_icon_path;
 bool Disable_built_in_translations;
 bool Weapon_shockwaves_respect_huge;
 bool Using_in_game_options;
+ubyte Arc_color_damage_p1_r = 64;
```

Comment:
I think it would be better to use a `std::tuple` for all these rgb value triples. That would make this variable declaration much shorter and keep the relevant values together in one variable.

Reply:
Changed as requested.